src/components/OrderableListV2/OrderableListV2.stories.tsx

```diff
@@ -0,0 +1,164 @@
+    [ItemDragState.Idle]: "tw-border-black-20",
+    [ItemDragState.Preview]: "tw-bg-white tw-border-violet-70 tw-border-2",
+};
+
+const ControlledTextArea = forwardRef<HTMLTextAreaElement, ControlledTextAreaProps>(({ value, onChange }, ref) => (
+    <textarea
+        ref={ref}
+        className="tw-w-full tw-block tw-border"
+        value={value}
+        onChange={(e) => onChange && onChange(e.target.value)}
+    />
+));
+ControlledTextArea.displayName = "ControlledTextArea";
+
+const Highlight: FC<HighlightProps> = ({ color, children }) => (
+    <span className={merge(["tw-font-medium", HighlightClasses[color]])}>{children}</span>
+);
+
+const storyItems: OrderableListItem<StoryListItem>[] = [
+    {
+        id: "1",
+        textContent: (
+            <p>
+                The list rendering is completely customizable through the &nbsp;
+                <Highlight color={HighlightColor.Green}>renderContent</Highlight> callback prop.
+            </p>
+        ),
+        alt: "one",
+        sort: 1,
+    },
+    {
+        id: "3",
+        textContent: (
+            <p>
+                Use the <Highlight color={HighlightColor.Green}>sort</Highlight> property in the &nbsp;
+                <Highlight color={HighlightColor.Green}>OrderableListItem</Highlight> to determine the position of items
+                items in the list.
+            </p>
+        ),
+        alt: "three",
+        sort: 2,
+    },
+    {
+        id: "4",
+        textContent: <p>Items can contain multiple focusable elements.</p>,
+        alt: "four",
+        sort: null,
+    },
+    {
+        id: "7",
+        textContent: (
+            <p>
+                The drag-preview is created as a new element, using the
+                <Highlight color={HighlightColor.Green}>renderContent</Highlight> callback with the dragged item key to
+                render its content. The state of any interactive items must be managed outside of the list to achieve
+                consistent rendering in the drag preview.
+            </p>
+        ),
+        alt: "seven",
+        sort: 3,
+    },
+];
+
+const renderContent = (
+    { textContent }: OrderableListItem<StoryListItem>,
+    { componentDragState, isFocusVisible }: DragProperties,
+) => {
+    return (
+        <div
+            className={merge([
+                "tw-break-word tw-border tw-border-black-40 tw-border-solid tw-p-3",
+                dragStoryStyles[componentDragState],
+                isFocusVisible && "tw-bg-violet-20",
+            ])}
+        >
+            <div className="tw-text-xs tw-text-black-60">Position: Foo</div>
+            <hr className="tw-mt-2 tw-mb-2 tw-border-black-20 tw-bg-black-20" />
+            <div>{textContent}</div>
+            <hr className="tw-mt-3 tw-mb-2 tw-border-black-20 tw-bg-black-20" />
+            <div className="tw-flex tw-justify-between tw-text-s">
+                <span className="tw-font-medium">{isFocusVisible && "Im in keyboard focus"}</span>
```

Comment:
isFocusVisible should be beside span element not inside it, or i'm missing something?

Reply:
Probably right (not sure as this is copy-pasted from the original OrderableList). Anyway, I moved it.